Tests/NautilusTelemetryTests/TraceExporterTests.swift

```diff
@@ -219,8 +215,30 @@ final class TraceExporterTests: XCTestCase {
 		let json = try exporter.exportOTLPToJSON(spans: [span1, span2], additionalAttributes: nil)
 
 		let normalizedJsonString = try XCTUnwrap(TestDataNormalization.normalizedJsonString(data: json, keyValuesToRedact: ["startTimeUnixNano", "spanId", "traceId", "resource"]))
-		let expectedOutput = #"{"resourceSpans":[{"resource":"***","scopeSpans":[{"scope":{"name":"NautilusTelemetry","version":"1.0"},"spans":[{"attributes":[{"key":"thread.name","value":{"stringValue":"main"}}],"events":[],"kind":1,"name":"root","spanId":"***","startTimeUnixNano":"***","status":{"code":1},"traceId":"***"},{"attributes":[{"key":"thread.name","value":{"stringValue":"main"}}],"events":[],"kind":1,"links":[{"spanId":"***","traceId":"***"}],"name":"hello","spanId":"***","startTimeUnixNano":"***","status":{"code":1},"traceId":"***"}]}]}]}"#
+		let expectedOutput = #"{"resourceSpans":[{"resource":"***","scopeSpans":[{"scope":{"name":"NautilusTelemetry","version":"1.0"},"spans":[{"attributes":[{"key":"thread.name","value":{"stringValue":"main"}}],"name":"root","spanId":"***","startTimeUnixNano":"***","traceId":"***"},{"attributes":[{"key":"thread.name","value":{"stringValue":"main"}}],"links":[{"spanId":"***","traceId":"***"}],"name":"hello","spanId":"***","startTimeUnixNano":"***","traceId":"***"}]}]}]}"#
+
+		XCTAssertEqual(normalizedJsonString, expectedOutput)
+	}
+
+	func testSpanLinkVariant1() throws {
+		let traceId1 = Identifiers.generateTraceId()
+		let traceId2 = Identifiers.generateTraceId()
+
+		// Test ok status
+		let span1 = Span(name: "root", traceId: traceId1, parentId: nil)
+		span1.recordSuccess()
+		// Test client kind + events
+		let span2 = Span(name: "hello", kind: .client, traceId: traceId2, parentId: nil, linkedParent: span1)
+		span2.addEvent("OMG")
+
+		let exporter = Exporter(timeReference: timeReference)
+		let json = try exporter.exportOTLPToJSON(spans: [span1, span2], additionalAttributes: nil)
+
+		let normalizedJsonString = try XCTUnwrap(TestDataNormalization.normalizedJsonString(data: json, keyValuesToRedact: ["startTimeUnixNano", "spanId", "traceId", "resource", "timeUnixNano"]))
+
+		let expectedOutput = #"{"resourceSpans":[{"resource":"***","scopeSpans":[{"scope":{"name":"NautilusTelemetry","version":"1.0"},"spans":[{"attributes":[{"key":"thread.name","value":{"stringValue":"main"}}],"name":"root","spanId":"***","startTimeUnixNano":"***","status":{"code":1},"traceId":"***"},{"attributes":[{"key":"thread.name","value":{"stringValue":"main"}}],"events":[{"name":"OMG","timeUnixNano":"***"}],"kind":3,"links":[{"spanId":"***","traceId":"***"}],"name":"hello","spanId":"***","startTimeUnixNano":"***","traceId":"***"}]}]}]}"#
 
+		print(normalizedJsonString)
```

Comment:
Nit: It can be nice to remove `print(…)` statements to keep test output focused.